README.md

```diff
@@ -1,57 +1,86 @@
-# iroha2-block-explorer-backend
+# Iroha 2 Blockchain Explorer Backend
 
+This readme provides an overview of the Iroha 2 blockchain explorer backend and instructions on how to install, run, and utilize the explorer's backend features.
+
+## Installation
+
+To set up the Iroha 2 blockchain explorer backend, follow these steps:
+
+1. Install Rust ```1.7.0```
```

Comment:
The latest version of Rust is [`1.70.0`](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html).

```suggestion
1. Install Rust `1.70.0`
```